packages/protocol/contracts/L1/SequencerRegistry.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "../common/EssentialContract.sol";
+import "./ISequencerRegistry.sol";
+
+/// @title SequencerRegistry
+/// A dummy implementation that only whitelist some trusted addresses. A real
+/// implementation would only allow a single proposer address to propose a block
+/// using some selection mechanism.
+/// @custom:security-contact security@taiko.xyz
+contract SequencerRegistry is EssentialContract, ISequencerRegistry {
```

Comment:
Sequencer => Proposer?

Reply:
I'm following the API spec from limechain for the naming here, potentially will be close to the actual name used sometime later.

Reply:
Should not we need (mayber dummy tho) `eligibleCountAt` and `sequencerByIndex`, `activate`, `register`, `withdraw`, `stake` functions too ? (Not sure if still needed, but have seen in the PoC proposal).